maskhit/train.py

```diff
@@ -255,18 +311,32 @@ def main():
 
     ########################################
     # prepare dataset
-    df_test = prepare_data(meta_split=meta_val,
-                           meta_file=meta_svs,
-                           vars_to_include=vars_to_include)
+    df_test = prepare_data(meta_split=meta_val,	
+                           meta_file=meta_svs,	
+                           vars_to_include=vars_to_include)	
+    df_train = prepare_data(meta_split=meta_train,	
+                            meta_file=meta_svs,	
+                            vars_to_include=vars_to_include)
 
-    df_train = prepare_data(meta_split=meta_train,
+    # check if train data is empty
+    if df_train.empty:

```

Comment:
Could you explain how df_train gets empty? I think the PreProcess should take care of setting up datasets. I'm also skeptical about adding train_test_split function here as that must have been done in the previous stage.